docs/sources/logql/metric_queries.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Metric queries
+weight: 20
+---
+
+# Metric queries
+
+Metric queries extend log queries by applying a function to log query results.
+This powerful feature creates metrics from logs.
+
+Metric queries can be used to calculate the rate of error messages or the top N log sources with the greatest quantity of logs over the last 3 hours.
+
+Combined with parsers, metrics queries can also be used to calculate metrics from a sample value within the log line, such as latency or request size.
+All labels, including extracted ones, will be available for aggregations and generation of new series.
+
+## Range Vector aggregation
```

Comment:
I think we should explain what "vectors" are; this is a term that may not be familiar to all, not least of which non-native English speakers.